.github/dependabot.yml

```diff
@@ -3,10 +3,14 @@ updates:
 
   - package-ecosystem: github-actions
     directory: /
+    labels: 
+    - auto-approve
```

Comment:
YOU CAN DO THAT?!

Reply:
And auto merge https://github.com/cdklabs/cdk-from-cfn/pull/235.